src/components/Launch/SchedulesTable.tsx

```diff
@@ -54,6 +57,18 @@ export const schedulesTableColumns: KeyedColumnProps[] = [
         label: 'frequency',
         width: schedulesTableColumnsWidths.frequency
     },
+    {
```

Comment:
Not exactly sure how this is used.

Reply:
This component isn't actually used at the moment. It's left over from the first iteration of the UI which had a very basic view of the schedules (both active and inactive) in a workflow.